documentation/changelogs/api/index.md

```diff
@@ -0,0 +1,10 @@
+# API changelogs
+
+Changelogs for <https://api.openverse.engineering/>.
+
+```{toctree}
+:glob:
+:titlesonly:
+
```

Comment:
```suggestion
:glob:
:reverse:
:titlesonly:


```

I _believe_ this is all we need to do to swap the order!

https://github.com/sphinx-doc/sphinx/pull/2527

Reply:
Yup! Should be.